tests/schema/test_scenario_validator.py

```diff
@@ -119,7 +118,6 @@ def setup_method(self):
                       "panel_count": 1,
                       "fit_to_limit": True,
                       "power_profile": "{\"filename\": \"Berlin_pv_1m.csv\"}",
-                      "cloud_coverage": 4,
```

Comment:
We like to keep the cloud coverage for the PV asset. Only for the general settings we like to remove it. So, this is not correct.